test/setup_transfer.cpp

```diff
@@ -869,7 +869,7 @@ int start_web_server(bool ssl, bool chunked_encoding, bool keepalive)
 	} while (ec);
 
 	char buf[200];
-	snprintf(buf, sizeof(buf), "python ../web_server.py %d %d %d %d"
+	snprintf(buf, sizeof(buf), "python2 ../web_server.py %d %d %d %d"
```

Comment:
hm.. I'm not sure about this. could finding the python executable be put in the python script perhaps? I take it your system doesn't have "python" in the path? (or is the problem that web_server.py is not python3 compatible?).

perhaps the python scripts should all start with "#!/usr/bin/env python". Fixing the python invocations could perhaps be a separate patch though